core/src/main/java/nu/yona/server/subscriptions/entities/Buddy.java

```diff
@@ -113,12 +120,20 @@ public Status getReceivingStatus()
 
 	public void setReceivingStatus(Status status)
 	{
-		getBuddyAnonymized().setReceivingStatus(status);
+		BuddyAnonymized buddyAnonymized = getBuddyAnonymized();
+		buddyAnonymized.setReceivingStatus(status);
+		BuddyAnonymized.getRepository().save(buddyAnonymized);
```

Comment:
No, JPA doesn't do that. I guess it just happened to occur elsewhere. It's better to do it explictely, as you do now.